src/main/java/com/andyinthecloud/githubsfdeploy/controller/GitHubSalesforceDeployController.java

```diff
@@ -613,40 +622,77 @@ private static void scanRepository(ContentsServiceEx contentService, RepositoryI
 			// Skip README.md (suffix overlaps with Custom Metadata!)
 			if(repo.getName().equalsIgnoreCase("readme.md"))
 				continue;
+			if (repo.getName().equalsIgnoreCase(".gitmodules")) {
```

Comment:
If we run into a `.gitmodules` file then we parse it to discover all the submodule definitions.